src/geopackage/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+cmake_minimum_required(VERSION 3.10)
+
+
+string(COMPARE EQUAL "${CMAKE_CXX_COMPILER_ID}" "IntelLLVM" _cmp)
+if (NOT _cmp)
+    message(WARNING "[NGen::geopackage] wkb.cpp cannot be optimized with "
+                    "${CMAKE_CXX_COMPILER_ID} due to a suspected optimizer/boost issue:\n"
+                    "https://github.com/NOAA-OWP/ngen/issues/567.\n"
+                    "Use IntelLLVM if optimization for this source file is required.")
+    # !! Required due to optimizer issue with either clang or
+    # !! boost::geometry::srs::transformation
+    set_source_files_properties(wkb.cpp PROPERTIES COMPILE_FLAGS -O0)
+endif()
+
+add_library(geopackage STATIC geometry.cpp
```

Comment:
Is there a reason to specify `STATIC`?

Reply:
No specific reason, I think I just copied what the geojson CMakeLists had. If it's better to put a different option then I'll change it to that!

Reply:
You can just drop it, and the build will produce whatever CMake has been configured to do in that build.